Sources/FluentSQL/DatabaseQuery+SQL.swift

```diff
@@ -89,3 +89,19 @@ extension DatabaseQuery.Sort {
         .custom(expression)
     }
 }
+
+extension FieldKey: SQLExpression {
+    public func serialize(to serializer: inout SQLSerializer) {
```

Comment:
Wouldn't it be more correct to write the `id`, `aggregate`, and `.string` cases out embedded in `SQLIdentifier` instead of raw?

Reply:
Hmm that's a good point. Yes definitely.

Reply:
Hm, that's gonna be tricky for a `.prefix` nested more than one level deep - and don't say "well that never happens right now anyway" even if it's true, 'cause it's tricky even for just a single level 😕. Not sure what the best way to deal with that is, actually, short of having something like `private func serializeRaw() -> SQLExpression { if case .prefix(let prefix, let key) = self { return SQLList([prefix.serializeRaw(), key.serializeRaw()], separator: SQLRaw("")) } else if case .id = self { return SQLRaw("id") } /* etc. */ }` and then implementing `serialize(to:)` as just `SQLIdentifier(self.serializeRaw()).serialize(to: &serializer)`... Does that work? Kinda off the top of my head here, and I mean obviously you'd just use a `switch`, not an `if`.